### ISSUE TITLE
Refactor / Consolidate Application Helper Methods

### ISSUE THREAD

**Issue by ajtran303** [ajtran303/monster_shop_2005#43]
```ruby 
# ApplicationController

### Judith
  def user
    if session[:user_id] != nil
      @user = User.find(session[:user_id])
    end
  end

### AJ
  def logged_in_user
    @logged_in_user ||= User.find(session[:user_id]) if session[:user_id]
  end

### Ruthie
  def current_user
    @current_user ||= User.find(session[:user_id]) if session[:user_id]
  end
```

So these can definitely be consolidated under one method signature.
We just have to decide on which one.
I think the `helper_method` keyword can narrow it down to two:

` helper_method :cart, :user, :current_user`

So let's vote! `user` or `current_user` ?

I have a branch checked out (pull login_logout remote into navigation local) and I'm ready to move on this.

**Comment by judithpillado** [comment]
I would like to use Ruthie's except instead of calling it current_user it could be called user. Unless it's referred to as "current_user" throughout the user stories, then we could just leave it as is.

**Comment by ajtran303** [comment]
@judithpillado Do you mean like this? 

```ruby
def user
    @user ||= User.find(session[:user_id]) if session[:user_id]
  end
```

I would be down with it.

**Comment by judithpillado** [comment]
Current user it is.

**Comment by ljwhite** [comment]
I just saw this...but I would vote current_user as well.

**Comment by ajtran303** [comment]
Closed by #49
